module/commands/regolamento_didattico.py

```diff
@@ -25,7 +25,22 @@
     'Regolamento Didattico 2015/2016_m': 'http://web.dmi.unict.it/sites/default/files/documenti_sito/Regolamento%20Didattico%20LM18%201516.pdf'
 }
 
-REGOLAMENTI = {'triennale': reg_doc_triennale, 'magistrale': reg_doc_magistrale}
+reg_doc_triennale_L35 = {
+    'Regolamento Didattico 2020/2021': 'http://web.dmi.unict.it/sites/default/files/files/L%2035_Matematica_2020_21approvato.pdf',
+    'Regolamento Didattico 2019/2020': 'http://web.dmi.unict.it/sites/default/files/files/L%2035_Matematica.pdf',
+    'Regolamento Didattico 2018/2019': 'http://web.dmi.unict.it/sites/default/files/files/Regol%20L35_1819%20approvato%20Senato.pdf',
+    'Regolamento Didattico 2017/2018': 'http://web.dmi.unict.it/sites/default/files/files/L%2035%20ufficiale1718.pdf',
+    'Regolamento Didattico 2016/2017': 'http://web.dmi.unict.it/sites/default/files/files/regolamento1617pubbldallAteneo.pdf',
+}
+
+reg_doc_magistrale_LM40 = {
+    'Regolamento Didattico 2020/2021_m': 'http://web.dmi.unict.it/sites/default/files/documenti_sito/LM%2040_Matematica%202020-21%20approvato%20Senato.pdf',
+    'Regolamento Didattico 2019/2020_m': 'http://web.dmi.unict.it/sites/default/files/documenti_sito/LM%2040_Matematica%2019-20%20approvato%20senato.pdf',
+    'Regolamento Didattico 2018/2019_m': 'http://web.dmi.unict.it/sites/default/files/documenti_sito/LM%2040%20Matematica%202018-19%20approvato%20senato.pdf',
+}
+
+REGOLAMENTI = {'triennale_L31': reg_doc_triennale_L31, 'magistrale_LM18': reg_doc_magistrale_LM18, 'triennale_L35': reg_doc_triennale_L35, 'magistrale_LM40': reg_doc_magistrale_LM40}
```

Comment:
to improve the **readability**:
```suggestion
REGOLAMENTI = {
  'triennale_L31': reg_doc_triennale_L31,
  'magistrale_LM18': reg_doc_magistrale_LM18,
  'triennale_L35': reg_doc_triennale_L35,
  'magistrale_LM40': reg_doc_magistrale_LM40
}
```